lib/coherence/controllers/unlock_controller.ex

```diff
@@ -49,25 +49,26 @@ defmodule Coherence.UnlockController do
       case LockableService.unlock_token(user) do
         {:ok, user} ->
           if user_schema.locked?(user) do
-            if Config.mailer?() do
-              send_user_email :unlock, user, router_helpers().unlock_url(conn, :edit, user.unlock_token)
-              put_flash(conn, :info, Messages.backend().unlock_instructions_sent())
-            else
-              put_flash(conn, :error, Messages.backend().mailer_required())
+            conn = case Config.mailer?() do
+              true ->
+                send_user_email :unlock, user, router_helpers().unlock_url(conn, :edit, user.unlock_token)
+                put_flash(conn, :info, Messages.backend().unlock_instructions_sent())
+              _ ->
+                put_flash(conn, :error, Messages.backend().mailer_required())
             end
-            |> redirect_to(:unlock_create, params)
+            conn
+            |> respond_with(:unlock_create_success, %{params: params, user: user})
           else
             conn
-            |> put_flash(:error, Messages.backend().your_account_is_not_locked())
-            |> redirect_to(:unlock_create_not_locked, params)
+            |> respond_with(:unlock_create_error_not_locked, %{params: params, error: Messages.backend().your_account_is_not_locked()})
```

Comment:
Same single pipe style guide comment. Also, I think there is a line length issue here.